RecoBTag/ONNXRuntime/plugins/DeepDoubleXONNXJetTagsProducer.cc

```diff
@@ -141,12 +146,33 @@ void DeepDoubleXONNXJetTagsProducer::produce(edm::Event& iEvent, const edm::Even
       for (const auto& taginfo : *tag_infos) {
         if (!taginfo.features().empty()) {
           make_inputs(idx, taginfo);
+          etas_.push_back(taginfo.jet()->eta()); //DEBUG
           ++idx;
         }
       }
 
       // run prediction
       outputs = globalCache()->run(input_names_, data_, output_names_, batch_size)[0];
+
+      // DEBUG: Dump inputs to file
```

Comment:
Has this debug to remain in the producer?

Reply:
I can gate it by a debug_ bool, but I would prefer to keep it here as it helps a lot with catching problems when syncing with the outside framework.